doc/source/myrocks/status_variables.rst

```diff
@@ -592,6 +596,14 @@ database.
 
 This variable isn't used anymore and will be removed in future releases.
 
+.. variable:: rocksdb_row_lock_deadlocks
+
+This variable shows the number of deadlocks.
```

Comment:
Maybe be a wee little more descriptive, like "This variable shows the total number of deadlocks that have been detected since the instance was started."